### DISCUSSION TITLE
Edit the edit link?

### DISCUSSION THREAD

**Discussion by christophsax** [quarto-dev/quarto-cli#5281] [Q&A] [answered]
In Quarto, there is the lovely feature that directly links to the edit page on GitHub. At first glance, this seems to work for other platforms, such as Gitea, as well. However, quarto points to `edit`, Gitea expects it at `_edit`.

Is there any way the edit link can be tweaked?

This is where Quarto links to: `.../edit/main/index.qmd`
This is how a Gitea edit link looks like: `.../_edit/main/index.qmd`

**Comment by mcanouil** [comment, marked as answer]
There is not currently a way to customise those links far as I know.
This could be feature request to choose which Git host is used instead/in addition of GitHub.

**Reply by mcanouil** [reply]
FYI, I opened an issue to track this for the future, see #5301 for follow-up.

**Comment by Javier-2712** [comment]
How can I customize my html documents so that the links open from a mobile phone? On my computer if you can, but not on the mobile. I leave an example of how they look:
https://javier-2712.github.io/Multivariados/Taller7.2_LDA/Taller7.2.lda.colibr%C3%ADes.html

**Reply by mcanouil** [reply]
Please open a new discussion when the question is actually not the same.

Also, I quite not understand what you want to do or try to have. What are we suppose to see or not see in your link?

**Reply by Javier-2712** [reply]
Good day. I need the text edited and generated in the html format not to be flat and that the links that I place can be executed from a cell phone. Currently it is only possible to access the links from a computer.

**Reply by mcanouil** [reply]
@Javier-2712 please as I said, open a new discussion.

PS: GitHub edits links are visible if you are logged in and if you setup properly this.

**Reply by Javier-2712** [reply]
This is the YAML that I use to edit the html documents. I don't understand why it won't let me enter links from my cell phone.
---
title: "name"
subtitle: "name"
author: "name"
date: 'Actualizado: `r Sys.Date()`'
lang: "es-ES"
format:
  html:
    toc: true
    toc-title: Contents
    toc-depth: 3
    toc-location: left
    html-math-method: katex
    css: styles.css  
    link-external-icon: true
---
